src/main/java/lottogame/domain/lotto/LottoNumber.java

```diff
@@ -1,93 +1,64 @@
 package lottogame.domain.lotto;
 
-import lottogame.utils.DuplicateLottoNumberException;
-import lottogame.utils.InvalidBonusBallNumberException;
 import lottogame.utils.InvalidLottoNumberRangeException;
 
-import java.util.ArrayList;
-import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
 
 public class LottoNumber {
-    private static final int LOTTO_MIN = 1;
-    private static final int LOTTO_MAX = 45;
-    private static final int LOTTO_NUMBER_VOLUME = 6;
-    private final List<Integer> numbers = new ArrayList<>();
-
-    public LottoNumber() {
-        for (int i = 0; i < LOTTO_NUMBER_VOLUME; i++) {
-            numbers.add(makeNumber());
+    public static final int LOTTO_NUMBER_MIN = 1;
+    public static final int LOTTO_NUMBER_MAX = 45;
+    private final int number;
+
+    private static final Map<Integer, LottoNumber> lottoNumbers = new HashMap<>();
```

Comment:
미리 만들어 놓는 방식이 좋네요 👍 👍 
```suggestion
    private static final Map<Integer, LottoNumber> LOTTO_NUMBERS = new HashMap<>();
```